cli_tools/common/utils/logging/service/literal_loggable_test.go

```diff
@@ -57,6 +57,8 @@ func TestSingleImageImportLoggable(t *testing.T) {
 	inflationTimeValue := int64(10000)
 	shadowInflationTimeValue := int64(5000)
 	matchResultValue := "true"
+	isUEFICompatibleImageValue := true
+	isUEFIDetectedValue := true
```

Comment:
since there are only 2 possible values, it's a good idea to test different combos of these. E.g. if there's a bug where isUEFICompatibleImageValue is used instead of isUEFIDetectedValue, this test would not detect it as both are true here.

Reply:
It's covered by unit test in "processor_test.go". I may move it to another file after refactoring.